flask_resty/pagination.py

```diff
@@ -188,21 +188,52 @@ def spec_declaration(self, path, spec, **kwargs):
 class CursorPaginationBase(LimitPagination):
     cursor_arg = 'cursor'
 
+    def ensure_query_sorting(self, query, view):
```

Comment:
I think it's cleaner if instead of returning a tuple `get_page` first calls `field_orderings  = get_field_orderings` and then `query = ensure_query_sorting()`

Reply:
it would be, but i couldn’t figure out a good way to tell which orderings  needed to be added in that case. we need to update both the query and the orderings, and that logic isn’t specific to the relay scheme